inc/Zenova/Platform.h

```diff
@@ -45,6 +47,8 @@ namespace Zenova {
 		static uintptr_t GetModuleBaseAddress(const std::wstring& modName);
 		static u32 GetModuleSize(const char*);
 		static void* LoadModule(const std::string& module);
+		static void* LoadModModuleAndResolveImports(const ModInfo& modInfo, const std::string& module);
```

Comment:
These should be in PlatformImpl as Zenova should realistically be the only one calling these functions

Reply:
ResolveModModuleImports also?